lib/sikuli/gsearch.sikuli/gsearch.py

```diff
@@ -40,18 +40,16 @@ def focus_search_inputbox(self):
     # this calculation is based on english version of google image search only
     def click_result_image(self, x, y):
         a = -120 + 280 * (x - 1)
-        b = 275 + 200 * (y - 1)
+        b = 190 + 200 * (y - 1)
```

Comment:
IMO, we should open an issue for tracking the x, y axis issues of Google Images Search.
Currently, it's okay for this PR.